en.dict.combodo-webhook-integration.php

```diff
@@ -62,6 +62,8 @@
 	'Class:ActionWebhook/Attribute:method/Value:put' => 'PUT',
 	'Class:ActionWebhook/Attribute:method/Value:patch' => 'PATCH',
 	'Class:ActionWebhook/Attribute:method/Value:delete' => 'DELETE',
+	'Class:ActionWebhook/Attribute:path' => 'Path',
+	'Class:ActionWebhook/Attribute:path+' => 'Additional path to add to the connection URL',
```

Comment:
```suggestion
	'Class:ActionWebhook/Attribute:path+' => 'Additional path to append to the connection URL',
```

What do you think of that small change? Would be clearer for people not that confortable with APIs IHMO.

Reply:
Sure, no problem.